SimPEG/electromagnetics/natural_source/receivers.py

```diff
@@ -29,318 +31,77 @@ class BaseRxNSEM_Point(BaseRx):
         {
             "real": ["re", "in-phase", "in phase"],
             "imag": ["imaginary", "im", "out-of-phase", "out of phase"],
+            "apparent resistivity": [
+                "apparent_resistivity",
+                "apparent-resistivity",
+                "app_rho",
+            ],
+            "phase": ["phi"],
         },
     )
 
-    def __init__(self, locs, orientation=None, component=None):
+    def __init__(
+        self,
+        locations=None,
+        orientation=None,
+        component=None,
+        locations_e=None,
+        locations_b=None,
```

Comment:
should we stick with `locations_h` here? Internally, we are working with `h`, so might be more consistent?

Reply:
I like consistency :), I'm game with changing to h. I can make these changes on my next push.